src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,13 +30,36 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that creates and lists comment data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   static final String comment_id = "comment-input";
   static final String redirect_location = "/comment.html";
   private List <String> comments;
+  
+  /** Responsible for creating new comments */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
+    Query query = new Query("Message").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
Here and throughout the file, use string constants, e.g. `public static final String DATASTORE_MESSAGE = "Message";` for these types of strings.

For small projects, it's ok to repeat these string comments `"timestamp"`, `"Message"`, `"comment"`, `"num-comments"`, etc. However, as projects gains more collaborators, the chances of a typo increases. These typos creates hard to track down bugs. Additionally, a descriptive string constant name improves the readability of the code.